src/components/menu/menu.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import styles from "./menu.module.scss";
+
+export const Menu = () => {
+  const menu = ["all", "documentary", "comedy", "horror", "crime"];
+  const sortOptions = ["Release date", "Title"];
+
+  const [selectedMenu, setSelectedMenu] = useState("all");
+  
+  return (
+    <div className={styles.menuWrapper}>
+      <nav className={styles.menuItems}>
+        {menu.map((item) => {
+            const active = item === selectedMenu
```

Comment:
Advise to always add `;` in code at the end of row, especially when you have formatters that are wrapping your code. It really simplifies reading. Let's add [this](https://eslint.org/docs/latest/rules/semi) rule to ESLint, please